_rules/element-lang-matches-default-off6ek.md

```diff
@@ -0,0 +1,337 @@
+---
+id: off6ek
+name: HTML element language subtag matches default language
```

Comment:
(trying to keep the discussion here, there are several comments related to this)
Good point. I have been copying the existing rule without too much thinking. I now agree that 3.1.2 should fail on parts with mixed languages and that 
```html
<p lang="en">
  The Dutch phrase "Hij ging met de kippen op stok" literally translates into "He went to roost with the chickens", but it means that he went to bed early.
</p>
```
is probably failing it.

3.1.2 has a few exceptions for some words, but I guess it is OK to just copy them in the expectation since this is a manual rule anyway…

Then we can certainly require all words to be in the same language as the `lang` attribute.

A (small) consequence is that 
```html
<p lang="en">Paul put dire comment on tape</p>
<p lang="fr">Paul put dire comment on tape</p>
```
will become 2 Passed targets, which is actually rather good, I guess…